src/ansi-c/CMakeLists.txt

```diff
@@ -5,9 +5,17 @@ add_executable(converter library/converter.cpp)
 
 file(GLOB ansi_c_library_sources "library/*.c")
 
-add_custom_command(OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/cprover_library.inc"
-    COMMAND cat ${ansi_c_library_sources} | $<TARGET_FILE:converter> > "${CMAKE_CURRENT_BINARY_DIR}/cprover_library.inc"
-    DEPENDS converter ${ansi_c_library_sources})
+# Convert cmake paths to native paths for Windows builds
+file(TO_NATIVE_PATH "${ansi_c_library_sources}"
```

Comment:
I don't think this really quite does what you want here; `ansi_c_library_sources` is a list not a single path. Also, without a second argument to store the result in this is just a no-op.

Reply:
I may be abusing the semantics of TO_NATIVE_PATH, but the effect of this invocation is to replace the path separators.  The second argument in in the second line below your comment.

I like the idea of scrapping cat.  The cbmc project has been getting it from various ports of linux utilities.  The cbmc build has traditionally been getting it from cygwin.  I am trying to avoid cygwin, and I'm using the version that comes with the git shell that is part of the visual studio installation git installation.  The problem is that different ports do different things with path separators.

Whatever you come up with, I'm going to apply the patch to the 5.12 distribution to build it.  I'm trying to build distributions patches of 5.12 for the major repositories (homebrew, debian, windows).  I'm trying to set up github actions to automate this.